geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -263,31 +263,22 @@ some `docker build`, and `docker-compose` commands.
 Main Changes
 ------------
 
-Environment variables are now stored in the env.default and env.project files.
-
-The migration process will do the following UI-related changes (visible in the html diffs):
+The migration process will changes the following things:
+1. Environment variables are now stored in the env.default and env.project files.
+2. .mako files are replaced by .tmpl, this means templating don't allow use of python anymore.
+In order to keep this behavior, it is needed to add a custom script in `script/config-eval-templates` and add it to the entrypoint into the config image
+3. Functionalities in the vars file (2.4) are now stored in the database (2.5) and manageable via the administration interface
+4. In the geoportal folder, there is `static` and `static_ngeo` sub-folders. Both are more restricted in their uses.
+`static` can be used everywhere and is visible by all images, while `static_ngeo` is now only used and visible for the webpack build.
```

Comment:
```suggestion
4. In the geoportal folder, there is `static` and `static_ngeo` sub-folders. Since version 2.5 the usage is stricter.
`static` can be used everywhere and is visible by all images, while `static_ngeo` can only be used in the webpack build.
```